src/main/bash/local_run_consul.sh

```diff
@@ -1,2 +1,5 @@
 #!/bin/bash
-consul agent -server -bootstrap-expect 1 -data-dir /tmp/consul -ui-dir `dirname $0`/../../../src/test/resources/consul_ui
+consul agent -server -bootstrap-expect 1 -data-dir /tmp/consul -ui-dir `dirname $0`/../../../src/test/resources/consul_ui -config-dir `dirname $0`/../../test/resources/consul_config &
```

Comment:
Can we create a 2nd bash script, maybe `travis_run_consul.sh`, that has the config for tokens? That way I can run without tokens normally, but for builds use the new script.